packages/stacks/src/stacks.utils.ts

```diff
@@ -71,3 +71,69 @@ export function getStacksBurnAddress(chainId: ChainID): string {
       return 'ST000000000000000000002AMW42H';
   }
 }
+
+function cleanHex(hexWithMaybePrefix: string): string {
+  if (hexWithMaybePrefix !== 'string') return hexWithMaybePrefix;
+  return hexWithMaybePrefix.startsWith('0x')
+    ? hexWithMaybePrefix.replace('0x', '')
+    : hexWithMaybePrefix;
+}
+
+export function hexToBuff(hex: string): Buffer {
+  return Buffer.from(cleanHex(hex), 'hex');
+}
```

Comment:
Do we still need this `Buffer` specific method versus a `UInt8Array` alternative?